xbmc/addons/IAddon.h

```diff
@@ -15,6 +15,20 @@
 
 class TiXmlElement;
 
+namespace KODI
+{
+namespace ADDONS
+{
+/*!
+ * @brief Identifier with 0 to select standard settings.xml from addon.
+ *
+ * All numbers higher as 0 relates to an independent addon instance where
+ * can have own settings.
```

Comment:
All numbers greater than 0 denote add-on instances with an individual set of settings.